frontend/src/stores/usePluginConfigurationStore.jsx

```diff
@@ -237,6 +237,21 @@ const usePluginConfigurationStore = create((set, get) => ({
       return null;
     }
   },
+  deletePlaybook: async (playbook) => {
+    let success = false;
+    try {
+      const response = await axios.delete(
+        `${PLAYBOOKS_CONFIG_URI}/${playbook}`,
+      );
+      success = response.status === 204;
```

Comment:
not sure about this "if", is it possible we get a different status code ? ex: 201